app/models/concerns/hyku_addons/account_behavior.rb

```diff
@@ -74,8 +75,24 @@ def switch_host!(cname)
         Rails.application.routes.default_url_options[:host] = cname
         Hyrax::Engine.routes.default_url_options[:host] = cname
       end
+
+      def setup_tenant_cache(is_enabled)
+        Rails.application.config.action_controller.perform_caching = is_enabled
+        ActionController::Base.perform_caching = is_enabled
+        # rubocop:disable Style/ConditionalAssignment
+        if is_enabled
+          Rails.application.config.cache_store = :redis_cache_store, { url: Redis.current.id }
+        else
+          Rails.application.config.cache_store = :file_store, Settings.cache_filesystem_root
+        end
+        # rubocop:enable Style/ConditionalAssignment
+        Rails.cache = ActiveSupport::Cache.lookup_store(Rails.application.config.cache_store)
+      end
+
+      def cache_api?
```

Comment:
If we are only using this method one it might be worth removing it and using the Flipflop directly.

Reply:
Might be useful in the future? Not sure it's big enough to worry about now